build/gen_stub.php

```diff
@@ -533,7 +533,12 @@ public function toOptimizerTypeMask(): string {
     }
 
     public function toEscapedName(): string {
-        return str_replace('\\', '\\\\', $this->name);
+        // Escape \u and \U to avoid compilation errors in generated macros
```

Comment:
Maybe rather something like this?

```suggestion
        // Escape backslashes, and also encode \u and \U to avoid compilation errors in generated macros
```